src/mlbstandings/app.py

```
@@ -0,0 +1,51 @@
+import flask
+
+import google.auth
+import logging
+
+# TODO figure out how to import these nicely and still have mypy work.
```

Pylint (3.3.7)

[warning] TODO figure out how to import these nicely and still have mypy work.

(W0511)